ArduSub/Sub.h

```diff
@@ -20,7 +20,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 // Header includes
 ////////////////////////////////////////////////////////////////////////////////
-
+#define CAMERA ENABLED
```

Comment:
No. That was a debug line so that the code of the camera would be built and I could confirm the build was compiling. Without this macro the code I did was dead code for the Sub